cost-updater/src/main/scala/org/broadinstitute/dsp/workbench/hamm/costUpdater/CostUpdaterService.scala

```diff
@@ -0,0 +1,45 @@
+package org.broadinstitute.dsp.workbench.hamm
+package costUpdater
+
+import cats.effect.Sync
+import cats.implicits._
+import fs2.concurrent.InspectableQueue
+import io.chrisdavenport.log4cats.Logger
+import io.circe.Encoder
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityEncoder._
+import org.http4s.dsl.Http4sDsl
+import CostUpdaterService._
+
+class CostUpdaterService[F[_]: Sync: Logger, A](queue: InspectableQueue[F, A]) extends Http4sDsl[F] {
```

Comment:
I'm confused about what the queue is for, could you add an explanation in the code?

Reply:
it's a queue passed to pubsub subscriber, which is used to hold items we pull from pubsub